src/utils/useTests.ts

```diff
@@ -1,19 +1,35 @@
+import { useGlobals } from "@storybook/manager-api";
 import { useCallback, useState } from "react";
 
-import { BrowserInfo, StoryTestFieldsFragment, TestMode } from "../gql/graphql";
+import { BrowserInfo, StoryTestFieldsFragment, TestMode, TestStatus } from "../gql/graphql";
 
 type BrowserData = Pick<BrowserInfo, "id" | "key" | "name">;
 type ModeData = Pick<TestMode, "name">;
 
+const useGlobalValue = (key: string) => {
+  try {
+    return useGlobals()[0][key];
```

Comment:
Is there something special about the first global that we use it as a store?

Reply:
`useGlobals` returns a tuple of `[value, setValue]`. I just want the value here.